source/Outfit.cpp

```diff
@@ -71,7 +71,7 @@ void Outfit::Load(const DataNode &node)
 		else if(child.Token(0) == "weapon")
 			LoadWeapon(child);
 		else if(child.Token(0) == "ammo" && child.Size() >= 2)
-			ammo = GameData::Outfits().Get(child.Token(1));
+			ammo = make_pair(GameData::Outfits().Get(child.Token(1)), 0);
```

Comment:
Let's add a comment here just so no one else gets curious like I did.

Reply:
Looks like this information isn't on the wiki either.